portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,35 +26,45 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private ArrayList<String> messages;
+  private Comments c;
```

Comment:
Single letter variable names are frowned upon pretty much everywhere except loop iterators because almost all code is read far more often than it's written.

I'd recommend changing `c` -> `comments` because it's a lot more descriptive and not that long.

Reply:
lol, I had too many things named `comments` so I tried doing something different. I'll come up with a more descriptive name though!